.bazelci/presubmit.yml

```diff
@@ -35,7 +35,11 @@ platforms:
     # - "//tests/integration:all"
     - "//examples:app"
     test_targets:
+    - "--"
     - "//tests/unit/..."
+    # TODO(jin): Concurrent download problems on macOS
+    # https://github.com/bazelbuild/rules_jvm_external/issues/54
+    - "-//tests/unit/kotlin/..."
```

Comment:
FYI: had to disable the test on the Mac too, due to network / race problems in https://github.com/bazelbuild/rules_jvm_external/issues/54

Reply:
Hmm maybe the CI can't resolve more than one maven_install repository concurrently?

Reply:
Yup, that's exactly it.